instance/models.py

```diff
@@ -43,7 +47,7 @@ def relate_with_persons_from_popit_api_instance(self, popit_api_instance):
         except Exception, e:
             self.do_something_with_a_vanished_popit_api_instance(popit_api_instance)
             return (False, e)
-        persons = Person.objects.filter(api_instance=popit_api_instance)
+        persons = PopoloPerson.objects.filter(api_instance=popit_api_instance)
```

Comment:
It seems like this line should probably not be changed until _after_ the data has been migrated in the later commits?